java/ql/test/library-tests/dataflow/threat-models/threat-models2.ext.yml

```diff
@@ -0,0 +1,8 @@
+extensions:
+extensions:
```

Comment:
Is this duplication of `extensions:` intentional?

Reply:
No - thank you again :-)